torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1018,8 +1018,21 @@ RegisterOperators reg(
              throw std::runtime_error(
                  "DictConstruct must have an even number of inputs");
            }
+           if (node->outputs().size() != 1) {
```

Comment:
This should be assertions. At this point we can assume the instruction is well-formed.